### PR TITLE
Add matid

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
Opened a pull request to include the license https://github.com/SINGROUP/matid/pull/5

**Comment by jan-janssen** [comment]
And another pull request for including `scikit-learn` - https://github.com/SINGROUP/matid/pull/6

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/chronic/time.patch [hunk 1]
```diff
@@ -0,0 +1,27 @@
+From c5f371adcd47a97a63e7edf3a5c92aee17ed5800 Mon Sep 17 00:00:00 2001
+From: Jan Janssen <jan-janssen@users.noreply.github.com>
+Date: Wed, 25 Nov 2020 21:29:31 +0100
+Subject: [PATCH] Fix time
```
Comment (dopplershift) [inline recipes/chronic/time.patch:4]: What's the rationale behind this patch?

Comment (jan-janssen) [reply recipes/chronic/time.patch:4]: `time.clock` was depreciated in Python 3.3 and has been removed in Python 3.8 https://github.com/python/cpython/pull/13270

**Pull request merged**